app/views/pages/cookies.html.erb

```diff
@@ -1,78 +1,181 @@
+            <td class="govuk-table__cell">_ga</td>
+            <td class="govuk-table__cell">These help us count how many people visit the service by tracking if you’ve visited before</td>
+            <td class="govuk-table__cell">2 years</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_gid</td>
+            <td class="govuk-table__cell">This randomly generated number is used to determine unique visitors to our site</td>
+            <td class="govuk-table__cell">24 hours</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">_gat_govuk_shared</td>
+            <td class="govuk-table__cell">This randomly generated number is used to reduce the number of requests made</td>
+            <td class="govuk-table__cell">2 years</td>
+          </tr>
+          </tbody>
+        </table>
+        <br>
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset" aria-describedby="cookie-consent-usage-hint">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xs">
+              Do you consent to cookies that measure website use?
+            </legend>
+            <div id="cookie-consent-usage-hint" class="govuk-hint">
+            </div>
+            <div class="govuk-radios govuk-radios--stacked">
+              <div class="govuk-radios__item">
+                <%= form.radio_button :cookie_consent_usage, true, class: "govuk-radios__input",
+                                      checked: cookies[:cookies_policy] && JSON.parse(cookies[:cookies_policy])['usage'] == true %>
+                <%= form.label(:cookie_consent_usage_1, "use cookies that measure my website use",
+                              class:"govuk-label govuk-radios__label", for:"cookie-consent-usage_yes") %>
+              </div>
+              <div class="govuk-radios__item">
+              <%= form.radio_button :cookie_consent_usage, false, class: "govuk-radios__input",
+                                      checked: cookies[:cookies_policy] && JSON.parse(cookies[:cookies_policy])['usage'] == false %>
 
-        <div class="panel panel-border-wide govuk-panel">
-          <p>GOV.UK cookies aren&rsquo;t used to identify you personally.</p>
-        </div>
-
-        <p>You&rsquo;ll normally see a message on the site before we store a cookie on your computer.</p>
-
-        <p>Find out more about <a rel="external" href="http://www.aboutcookies.org/">how to manage cookies.</a></p>
-
-        <h2 class="govuk-heading-m">How cookies are used on the Trade Tariff</h2>
-
-        <div class="panel panel-border-wide govuk-panel">
-          <p>This cookies policy is an additional policy that applies to the Trade Tariff service. The main <a href="https://www.gov.uk/help/cookies">GOV.UK cookies policy</a> also applies.</p>
+                <%= form.label(:cookie_consent_usage_2,  "No, do not use cookies that measure my website use",
```

Comment:
Feels like JSON.parse(cookies[:cookies_policy])['usage'] == false should stay on a helper method maybe? Say inside cookies_helper.rb.